src/common/eeprom.cpp

```diff
@@ -878,6 +856,47 @@ int8_t eeprom_test_PUT(const unsigned int bytes) {
 
 ///////////////////////////////////////////////////////////////////////////////
 /// Sheets profile methods
+static const float Z_OFFSET_MIN = -2.0F;
+static const float Z_OFFSET_MAX = 0.0F;
+
+float eeprom_get_z_offset() {
+    uint8_t index = variant8_get_ui8(eeprom_get_var(EEVAR_ACTIVE_SHEET));
+    if (index >= MAX_SHEETS)
+        return 0.F;
+
+    float value;
+#if (EEPROM_FEATURES & EEPROM_FEATURE_SHEETS)
+    Sheet sheet;
+    eeprom_get_var(static_cast<enum eevar_id>(EEVAR_SHEET_PROFILE0 + index), &sheet, sizeof(sheet));
+    value = sheet.z_offset;
+#else
+    eeprom_get_var(EEVAR_ZOFFSET_DO_NOT_USE_DIRECTLY, &value, sizeof(value));
+#endif
+    if (!isfinite(value))
+        return 0.F;
+    return std::clamp(value, Z_OFFSET_MIN, Z_OFFSET_MAX);
+}
+
+bool eeprom_set_z_offset(float value) {
```

Comment:
This function has return value, but it is not checked in any place where it is used.

Reply:
added return value checks